scripts/run_test262_suite.py

```
@@ -434,170 +490,328 @@ def evaluate_suite(
-
-                    summary["passed"] += tr_passed
-                    summary["failed"] += tr_failed
-
-                    # Build failed-test ID set from TestRunner output
-                    failed_set: set[str] = set()
-                    failed_messages: dict[str, str] = {}
-                    for name in tr_failed_tests:
-                        # Case 1: 'Test "<desc>" in suite "test262: <test_id>"'
-                        m = re.search(
-                            r'in suite "test262:\s*(.+?)"', name
+                    process = subprocess.run(
```

Ruff (0.15.11)

[error] `subprocess` call: check for execution of untrusted input

(S603)